examples/cfg.yaml

```diff
@@ -88,6 +88,10 @@ LOG_THRESHOLD: INFO
 
 
 # Disable the specified displays.
+# In this example, HDMI-1 will always be disabled, and eDP-1 will be disabled
+# only if both DP-1 and DP-2 are plugged.
```

Comment:
Can we please note that an explicit disabled will override the condition?
`LG Display 0x05EF` is always disabled in this test case. I reckon that's sensible behaviour.
```yaml
DISABLED:
  - NAME_DESC: LG Display 0x05EF
    IF:
    - UNPLUGGED:
      - XG349C
  - LG Display 0x05EF
```